juniorguru_chick/lib/intro.py

```diff
@@ -75,10 +75,33 @@ def generate_intro_message() -> dict[str, Any]:
         '- Záznamy přednášek? <#788822884948770846>\n'
         '- Něco jiného? <#769966887055392768> snese cokoliv\n'
         '- Nevíš, jak to tady funguje? Ptej se v <#806215364379148348>'
+    )
+
+    # TODO - replace with Czech version and reformat the string
+    gh_connection_snippet = """
+        '\n\n'
+        It looks like you've mentioned your GitHub profile in your intro message.
+        Would you like to connect your GitHub account to your Discord account? This will allow you to get a special role and access to a private channel. So very fun!
+
+        Here's how to do it:
+        1. Go to Discord settings.
+        2. Navigate to Connections.
+        3. Click on GitHub.
+        4. Authorize the connection.
+
+        If you have any questions, feel free to ask in the #ahoj channel. We're here to help!
+    """
+
+    if intro_message_content.find("github.com/") != -1:
```

Comment:
And then Python said: „Hold my beer!“
```suggestion
    if "github.com/" in intro_message_content:
```